docs/examples/actuators-cli.md

```diff
@@ -0,0 +1,94 @@
+# Actuators via ROS 2 Command Line
+
+This page describes how to use the the Create® 3 robot's actuators using the ROS 2 command line tools.
+
+## Audio
+
+### Play a sequence of notes
+
+Good times:
+```sh
+ros2 topic pub /cmd_audio irobot_create_msgs/msg/AudioNoteVector "{append: false, notes: [{frequency: 392, max_runtime: {sec: 0,nanosec: 177500000}}, {frequency: 523, max_runtime: {sec: 0,nanosec: 355000000}}, {frequency: 587, max_runtime: {sec: 0,nanosec: 177500000}}, {frequency: 784, max_runtime: {sec: 0,nanosec: 533000000}}]}" -1
+```
+
+Bad times:
+```sh
+ros2 topic pub /cmd_audio irobot_create_msgs/msg/AudioNoteVector "{append: false, notes: [{frequency: 369, max_runtime: {sec: 0,nanosec: 355000000}}, {frequency: 300, max_runtime: {sec: 0,nanosec: 533000000}}]}" -1
+```
+
```

Comment:
Do we want to include the action to play a repeating sequence of notes?

Reply:
I don't think that is necessary, but we can.